rest/route/middleware.go

```diff
@@ -447,7 +447,7 @@ func (p *RequiresProjectViewPermission) ServeHTTP(rw http.ResponseWriter, r *htt
 		Resource:      projectID,
 		ResourceType:  evergreen.ProjectResourceType,
 		Permission:    evergreen.PermissionTasks,
-		RequiredLevel: int(evergreen.TasksView),
+		RequiredLevel: int(evergreen.TasksView.Value),
```

Comment:
You don't need these int type assertions anymore now that the type of the variable is int